lib/src/components/buttons/details_button.dart

```diff
@@ -34,16 +39,27 @@ class DetailsButton extends StatelessWidget {
           children: [
             Text(
               title,
-              style: const TextStyle(
-                letterSpacing: -0.9,
-                fontSize: 19.2,
-                fontWeight: FontWeight.w500,
-              ).copyWith(
+              style: TextStyle(
+                fontFamily: 'Roboto',
```

Comment:
The string should probably be in some config class.
```suggestion
                fontFamily: FontConfig.family,
```